setup.rst

```diff
@@ -122,22 +123,161 @@ working only on pure Python code the pydebug build provides several useful
 checks that one should not skip.
 
 
-.. _build-dependencies:
+.. _unix-compiling:
 
-Build dependencies
-------------------
+UNIX
+----
 
-The core CPython interpreter only needs a C compiler to be built; if
-you get compile errors with a C89 or C99-compliant compiler, please `open a
-bug report <https://bugs.python.org>`_.
-However, some of the extension modules will need development headers
+The core CPython interpreter only needs a C compiler to be built,
+however, some of the extension modules will need development headers
 for additional libraries (such as the ``zlib`` library for compression).
 Depending on what you intend to work on, you might need to install these
 additional requirements so that the compiled interpreter supports the
 desired features.
 
-UNIX
-----
+If you want to install these optional dependencies, consult the
+:ref:`build-dependencies` section below.
```

Comment:
Is the terminology correct here (and elsewhere)?
AFAIU the _dependencies_, are required to build _extension modules_ (or simply _extensions_).  The dependencies can be _installed_ (by using ``yum``/``apt``/etc) and _built_ (indirectly, by ``make``).
The section below covers _installing_ for Linux, and both _installing_ and _building_ (since different flags are required by ``configure``) for macOS.